### DISCUSSION TITLE
Using markdown in copyright or announce vars in mkdocs.yml

### DISCUSSION THREAD

**Discussion by Andygol** [squidfunk/mkdocs-material#6098] [Q&A]
I use next code to localize Announcement

```jinja
{% extends "base.html" %}

{% block announce %}
  <div markdown>
    {{ config.announce }}
  </div>
{%- endblock %}
```

Localization is made via `mkdocs-static-i18n`, so I have `announce` var in the `mkdocs.yml`. 

I observed one frustrating aspect: the values of the `announce` or `copyright` variables must be in a plain HTML format.

The presence of `<div markdown>` together with the `md_in_html` plugin does not result in the conversion of markdown markup into HTML.

Is there any way to make it possible to write text in markdown in the values of variables (in `mkdocs.yml`) and then it is displayed correctly formatted in the copyright or/and announce blocks?

**Comment by facelessuser** [comment]
While I don't think this is possible out of the box (someone can correct me if I'm wrong), I imagine a MkDocs plugin could be crafted to alter the template (or maybe the final page output) by scanning for such blocks and converting them from Markdown to HTML via a `on_post_template` event (or maybe `on_post_page`). I imagine some experimentation would be required to figure out exactly when the desired content is available. The plugin would likely need to read in the `markdown_extension` plugin config and create its own Markdown process for conversion, but it is probably doable.
